README.md

```diff
@@ -1 +1,5 @@
-# Review-project-from-trials
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
```

Comment:
You should add author section

Reply:
- [ ]  Always make sure to include an explanation, a "why", for each requested change, and use polite words. Kindly edit and submit the project.